rapidjson/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+if (BUILDING_LEATHERMAN)
+    set(include_dir "${PROJECT_SOURCE_DIR}/vendor/rapidjson/include")
+    set(${include_var} ${include_dir} PARENT_SCOPE)
+
+    if(LEATHERMAN_INSTALL)
+        install(DIRECTORY "${include_dir}" DESTINATION "include/leatherman/vendor/rapidjson")
```

Comment:
You should try `make install` in the leatherman project to see what you end up with.